ingestion/src/metadata/utils/aws_client.py

```diff
@@ -17,6 +17,16 @@
 from metadata.config.common import ConfigModel
 
 
+class GlueClient:
+    def __init__(self, client) -> None:
+        self.client = client
```

Comment:
uh true, we need to generate classes for the singledispatch. Instead of a naked class we could use a pydantic model or a dataclass, otherwise linters will complain that this class has not enough methods and attributes.